h2o-core/src/main/java/water/util/VecUtils.java

```diff
@@ -382,11 +402,56 @@ public static Vec categoricalDomainsToNumeric(final Vec src) {
       }.doAll(Vec.T_NUM, src).outputFrame().anyVec();
   }
 
+  public static class CollectDoubleDomain extends MRTask<CollectDoubleDomain> {
```

Comment:
Just wondering: would it have any value to have a function on a Chunk to collect the unique values?